Makefile

```diff
@@ -62,21 +62,16 @@ ensure_ns:
 run_local_tests: libdds-build
 	python3 -m unittest discover ${VERBOSE_TEST}
 
+# TODO: Use a more straightforward approach than $@ - we no longer need it here.
 curl_local_URL := http://localhost:5000/api/dds-table/
 curl_prod_URL  := https://dds.prod.globalbridge.app/api/dds-table/
 CURL_URL       ?= ${$@_URL}
 
-# Slightly unusual approach in order to keep the command-line output legible.
-# May need to add "--insecure" when we add more targets.
+# NOTE: We may need to add "--insecure" when we add more targets.
```

Comment:
This would be for a local cluster. But I can now see two better solution than --insecure.
* A local cluster uses http.
* We provide self signed certificate to curl using --cacert argument.

Reply:
Comment updated.